packages/core/src/subscriptions/redux/actionTypes.js

```diff
@@ -38,6 +38,16 @@ export const PUT_USER_SUBSCRIPTIONS_SUCCESS =
 export const RESET_SUBSCRIPTIONS =
   '@farfetch/blackout-core/RESET_SUBSCRIPTIONS';
 
+/** Action type dispatched when the unsubscribe all subscriptions request fails. */
```

Comment:
```suggestion
/** Action type dispatched when the unsubscribe from subscription request fails. */
```

Reply:
(same for the remaining comments on this file)